app/controllers/schools/choose_programme_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Schools::ChooseProgrammeController < Schools::BaseController
+  skip_after_action :verify_authorized
+  skip_after_action :verify_policy_scoped
```

Comment:
Are we sure we want to skip them?

Or should we have a policy for this?

Reply:
I guess we could have `SchoolCohortPolicy`. It should specify you can only create for your own school

Reply:
Decided it's not worth the time - we never let the user specify which school they are choosing for, so there is no need to restrict that.